src/test/acceptance/features/email-address.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Enter email address
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to enter my email address
+
+  Background:
+    Given I have entered my details up to the email address page
+
+  Scenario: Enter in a valid email address
+    When I enter a valid email address
+    Then I am shown the check details page
+
+  Scenario: Do not enter in an email address
+    When I do not enter an email address
+    Then I am informed that the email address is in the wrong format
```

Comment:
Shouldn't this be `Then I am informed that I need to enter an email address`, hence different to the wrong format message?

Reply:
How about if this is renamed 'I am informed that I must enter an email address in the correct format'

Reply:
Renamed. Although we may need a discussion between the lack of consistency. Phone number has separate errors for missing and invalid number, nino and email only have one message to cover missing and invalid.